### DISCUSSION TITLE
Welcome to CVXPY Discussions!

### DISCUSSION THREAD

**Discussion by akshayka** [cvxpy/cvxpy#1269] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions.
  * Share ideas.
  * Engage constructively with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

On this thread, feel free to introduce yourself and share stories about how you use or contribute to CVXPY.

**Comment by josephjacks** [comment]
Glad to see this! Is there a discussion around the OSS solvers that exist in CO? Listing and comparing tradeoffs, etc?

**Reply by SteveDiamond** [reply]
That's a great idea! I'll create one.

**Comment by hritikshah345** [comment]
UserWarning: Your problem has too many parameters for efficient DPP compilation. We suggest setting 'ignore_dpp = True'.
How should I set 'ignore_dpp = True' in my Class ReluLayer? 
And also Is it possible to use CVXPY layer inside Deep neural network model that is runned in GPU

class ReluLayer(torch.nn.Module):
    def __init__(self, D_in, D_out):
        super(ReluLayer, self).__init__()
        self.W = torch.nn.Parameter(1e-3*torch.randn(D_out, D_in))
        self.b = torch.nn.Parameter(1e-3*torch.randn(D_out))
        z = cp.Variable(D_out)
        Wtilde = cp.Variable((D_out, D_in))
        W = cp.Parameter((D_out, D_in))
        b = cp.Parameter(D_out)
        x = cp.Parameter(D_in)
        prob = cp.Problem(cp.Minimize(cp.sum_squares(z-Wtilde@x-b)), [z >= 0, Wtilde==W])
        assert prob.is_dcp(dpp=True)
        self.layer = CvxpyLayer(prob, [W, b, x], [z])

    def forward(self, x):
        # when x is batched, repeat W and b 
        if x.ndim == 2:
            batch_size = x.shape[0]
            return self.layer(self.W.repeat(batch_size, 1, 1), self.b.repeat(batch_size, 1), x)[0]
        else:
            return self.layer(self.W, self.b, x)[0]
